File: FeedAPI/FeedAPI/Common/EntityFramework/Models/Asset.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.EntityFramework.Models
{
    [Table("assets")]
    public class Asset
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("dealid")]
        public int DealId { get; set; }

        [Column("imagename")]
        public string ImageName { get; set; }

        [Column("imagedata")]
        public byte[] ImageData { get; set; }

        public Asset() { }

        public Asset(int id, int dealId, string imageName, byte[] imageData)
        {
            Id = id;
            DealId = dealId;
            ImageName = imageName;
            ImageData = imageData;
        }
    }
}

File: UI/Client/src/app/Models/Asset.ts
export class Asset{
    id: number;
    deallId: number;
    imageName: string;
    imageData: string;


    constructor(
        _id: number,
        _deallId: number,
        _imageName: string,
        _imageData: string,
        )        {
        this.id = _id;
        this.deallId =_deallId;
        this.imageName = _imageName;
        this.imageData = _imageData;
    }
}